packages/docs/components/molecules/StickyTableOfContent/stories/app.twig

```diff
@@ -0,0 +1,32 @@
+<style type="text/css">
+  .current { color: purple; }
+</style>
+
+<div data-component="StickyTable" class="flex items-start relative">
+  <div class="bg-blue-200 w-1/3 sticky flex flex-col top-4">
+    <a href="#item-1" data-component="StickyTableItem" class="py-2 px-2">item-1</a>
+    <a href="#item-2" data-component="StickyTableItem" class="py-2 px-2">
+      item-2
+    </a>
+    <a href="#item-3" data-component="StickyTableItem" class="py-2 px-2">item-3</a>
+  </div>
+  <div data-component="StickyTableSection" class="w-2/3">
+    <div class="relative bg-purple-200 h-[300px]">
+      <div data-ref="item[]" id="item-1" class="bg-pink-200 h-[50px]">
```

Comment:
**suggestion :** The `SticktyTableSection` component could be used for each section instead of using refs

The component could then simply use `withMountWhenInView` or `withIntersectionObserver` to communicate its state to the `StickyTable` parent.